install_build_dependencies.sh

```diff
@@ -211,7 +211,7 @@ fi
 current_cmake_ver=$($cmake_command --version | sed -ne 's/[^0-9]*\(\([0-9]\.\)\{0,4\}[0-9][^.]\).*/\1/p')
 required_cmake_ver=3.20.0
 if [ ! "$(printf '%s\n' "$required_cmake_ver" "$current_cmake_ver" | sort -V | head -n1)" = "$required_cmake_ver" ]; then
-    installed_cmake_ver=3.23.2
+    installed_cmake_ver=3.24.0
```

Comment:
this script is not used during `.ci/openvino-onnx/Dockerfile` build. 
Options:
- use this script to install dependencies instead of manual  apt-get commads
- copy cmake installtion related code from this script to that Dockerfile

Reply:
Seems like the CMake had been inherited after build OpenVINO, the compiler process had success.